### ISSUE TITLE
Error `Twig\Environment::clearCacheFiles` method is deprecated

### ISSUE THREAD

**Issue by alilleybrinker** [timber/timber#2340]
## Expected behavior

On a default, fresh Timber installation (fresh Wordpress installation as well), I don't expect there to be errors.

## Actual behavior

With `WP_DEBUG` set to `true`, whitespace appears at the top of the admin panel because the `.php-error` class has been applied to the sidebar, although no error appears. Modifying `admin-header.php` to write the last error to the error log shows the error is that Timber is using the method `Twig\Environment::clearCacheFiles` which is marked as deprecated.

## Steps to reproduce behavior

1. Install Wordpress.
2. Install Timber (latest version) from plugin directory.
3. Turn on `WP_DEBUG` in your `wp-config.php`.
4. Observe the presence of a bar of whitespace between the top bar and the main content of the admin panel
5. Place an `error_log` of the `last_error` in `wp-admin/admin-header.php` (right above where the `php-error` class is assigned, to see the error printed out in the default log location `wp-content/debug.log`.
6. Open the file to observe the error as described.

## What version of WordPress, PHP and Timber are you using?

Wordpress: 5.5.1
PHP: 7.4.11
Timber: 1.18.2

## How did you install Timber? (for example, from GitHub, Composer/Packagist, WP.org?)

From the plugin search within my Wordpress installation.

**Comment by alilleybrinker** [comment]
Looks like `Loader::clear_cache_twig` is the offending method: https://github.com/timber/timber/blob/master/lib/Loader.php#L238-L249

**Comment by jarednova** [comment]
Thanks so much for the issue and identifying the problem point @alilleybrinker! Is there an obvious fix? I want to give you first crack at the PR opportunity.

**Comment by alilleybrinker** [comment]
Not sure there _is_ a good option if you're set on sticking with Twig `1.x`. Per the Twig documentation, these cache-clearing options are all deprecated: https://twig.symfony.com/doc/1.x/deprecated.html#miscellaneous

Twig is currently on 3.x. I haven't looked in detail to see what's changed.

**Comment by gchtr** [comment]
Just a quick note that as of Timber 2.x, we will drop support for Twig 1.x. Timber will require a Twig  version >= 2.10: https://github.com/timber/timber/blob/2.x/composer.json#L32.

I guess testing support for Twig 3.x is another taks we will have to work on.

**Comment by alilleybrinker** [comment]
The errors from Twig are being produced by [`trigger_error`](https://www.php.net/manual/en/function.trigger-error.php) and so could be handled via [`set_error_handler`](https://www.php.net/manual/en/function.set-error-handler.php) and [`restore_error_handler`](https://www.php.net/manual/en/function.restore-error-handler.php) with a modification to `Loader::clear_cache_twig` which throws away the error. May be a decent bandage, as the method works but is deprecated and Timber is moving to a newer version anyway.

**Comment by jarednova** [comment]
It might be that "bandage" is all we're able to do for plugin users who we'll soon leave behind. @alilleybrinker is it as simple as ...

```php
if ( method_exists($twig, 'clearCacheFiles')) {
	set_error_handler("handle_cache_deprecation_warnings");
	$twig->clearCacheFiles();
	restore_error_handler();
}
```
```php
function handle_cache_deprecation_warnings() {
// do nothing
}
```
(and yes, sorry for a big gap here, just finished work for the year and closing up as much Timber stuff as I can over the holidays)

**Comment by gchtr** [comment]
This should be fixed in 2.x. It was resolved in #2611. The `clearCacheFiles` was replaced with `rmdir`  (see https://github.com/timber/timber/commit/81eca24a3a05ea9ea00bb82016d1cc3f49341c75#diff-1314d7908e6167b9fbe3d52c6f0c3cadf76637c75eca9c9c36d66106c579edd3L555).
